lib/smart_proxy_remote_execution_ssh/runners/polling_script_runner.rb

```diff
@@ -36,12 +36,12 @@ def prepare_start
     def initialization_script
       close_stdin = '</dev/null'
       close_fds = close_stdin + ' >/dev/null 2>/dev/null'
-      main_script = "(#{@remote_script} #{close_stdin} 2>&1; echo $?>#{@base_dir}/init_exit_code) >#{@base_dir}/output"
+      main_script = "(#{@remote_script_wrapper} #{@remote_script} #{close_stdin} 2>&1; echo $?>#{@base_dir}/init_exit_code) >#{@base_dir}/output"
       control_script_finish = "#{@control_script_path} init-script-finish"
       <<-SCRIPT.gsub(/^ +\| /, '')
       | export CONTROL_SCRIPT="#{@control_script_path}"
       | sh -c '#{main_script}; #{control_script_finish}' #{close_fds} &
-      | echo $! > '#{@base_dir}/pid'
+      | echo $! > /dev/null 
```

Comment:
Why write it at all? We could just discard this line completely

Reply:
I thought that it wouldn't work without that. But I was wrong, it didn't work because I also removed `&`.